### ISSUE TITLE
`wp vp clone` should work like `git clone`

### ISSUE THREAD

**Issue by JanVoracek** [versionpress/versionpress#1045]
We are constantly trying to have the WP-CLI commands semantically very close to Git. Commands `wp vp pull` and `wp vp push` work like Git commands + they update the database. However, the `wp vp clone` command works differently. 

When you run `git clone <repository>` it clones a remote repository into your location. Our `clone` works more like `fork`. It clones a repository you are standing in to another one. Hence, we should rename it to `wp vp fork` or not use it at all. Running `wp vp clone <repository>` should clone a remote repository and setup the site.

**Comment by unclhos** [comment]
I would like to have the fork command, I use the current clone command.

**Comment by borekb** [comment]
I'm torn on this. On the one hand, I agree that `wp vp clone https://...` should work similarly to Git, as Jan suggests. On the other hand, _cloning_ is a term commonly used by WP hosts and tools to actively create another environment, so the current `wp vp clone --name=staging` command also feels quite natural. The word "fork" sounds overly technical and I'm also thinking how to write documentation about it – I'd very much like to avoid something like:

> To clone the site, use the `fork` command. Don't use the `clone` command because...

Some option would be to make `wp vp clone` work in both directions and perhaps rename the `--name` attribute to a more explicit `--to`. For example, `wp vp clone https://...` would set up a local site and `wp vp clone --to=staging` would create a staging clone. But I'm not sure about this either :)

Another option: name the "fork" command `clone-to`. For example, `wp vp clone-to staging`.

**Comment by unclhos** [comment]
If you make a real "wp vp clone", will probably need to create a mantain a WP-CLI package so you don't have the plugin files available locally. But giving that some thought. That would be excellent as well so for people that use VP a lot don't have to use the --require flag when working with a non-active site.

**Comment by unclhos** [comment]
Also for documentation, you can just change your wording a bit

> To duplicate this site, use the fork command. To duplicate a remote site, use the clone command.

**Comment by borekb** [comment]
> ... create a mantain a WP-CLI package ... That would be excellent as well so for people that use VP a lot don't have to use the --require flag when working with a non-active site.

Yes we're thinking about that. There's probably not an issue for it yet but definitely on our minds.
